src/utils/service-requests/playlist-services.js

```diff
@@ -0,0 +1,85 @@
+import axios from "axios";
+
+const getPlaylists = async (dataDispatch) => {
+  try {
+    const token = localStorage.getItem("token");
+    const {
+      data: { playlists },
+    } = await axios.get("/api/user/playlists", {
+      headers: { authorization: token },
+    });
+    dataDispatch({ type: "GET_PLAYLISTS", payload: playlists });
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+const createPlaylist = async (title, dataDispatch) => {
+  try {
+    const token = localStorage.getItem("token");
```

Comment:
store the localstorage token in a variable in your auth call and use it everywhere else rather than checking localstorage before every call